webauthn-server-core/src/test/scala/com/yubico/webauthn/RelyingPartyAssertionSpec.scala

```diff
@@ -66,14 +69,16 @@ import scala.util.Success
 import scala.util.Try
 
 @RunWith(classOf[JUnitRunner])
-class RelyingPartyAssertionSpec extends FunSpec with Matchers with ScalaCheckDrivenPropertyChecks {
+class RelyingPartyAssertionSpec extends FunSpec with Matchers with ScalaCheckDrivenPropertyChecks with BeforeAndAfterAll {
 
   private def jsonFactory: JsonNodeFactory = JsonNodeFactory.instance
-  private val crypto = new BouncyCastleCrypto()
+  private val crypto = new Crypto()
 
   private def sha256(bytes: ByteArray): ByteArray = crypto.hash(bytes)
   private def sha256(data: String): ByteArray = sha256(new ByteArray(data.getBytes(Charset.forName("UTF-8"))))
 
+  override def beforeAll() = Security.addProvider(new BouncyCastleProvider())
```

Comment:
Those who want to use Bouncy Castle should do this or other equivalent (e.g., by editing `$JAVA_HOME/lib/security/java.security` or passing command line flags).